Etabs_Adapter/CRUD/Create/SectionProperty.cs

```diff
@@ -238,5 +241,167 @@ private void SetProfile(IProfile profile, string sectionName, IMaterialFragment
         }
 
         /***************************************************/
+
+
+        private bool GetFromDatabase(ISectionProperty bhSection)
```

Comment:
Might call this something like `LoadFromDatabase` instead of get, as "get" sounds a bit more like reading something, while this is creating.